fixes #36: improve zero-to-one section, add compile CLI command (#37)
stackrlabs/docs: docs/pages/build/zero-to-one/run-it.mdx

```ts showLineNumbers [index.ts]
const rollup = await MicroRollup({
  config: stackrConfig,
  actionSchemas: [ActionSchema1, ActionSchema2, ...],
  stateMachines: [machine],
  ...
});

await rollup.init() // [!code focus]
```

::::

## Let's Run 🏃‍♂ [Sandbox mode]

To be doubly sure that everything is working fine, let's try to run your application and see if it works as expected.

In this example, we will run our micro-rollup in sandbox mode.

:::info
**Sandbox mode** 🧪

Sandbox mode doesn't require registration or deployment of your MRU. The SDK creates a temporary SQLite database to store data and the blocks are not sent to Vulcan or L1 for verification.
It's recommended to use this mode while development and for experimentation.

> :::

Reviewer: add note on to enable or check if the MRU is in sandbox mode, make sure `isSandbox` in stackrConfig is set to `true`

### Run using Node.js 

:::code-group

```bash [npm]
npm start
```
```bash [bun]
bun start
```

:::

### Run using Docker 🐳

- Build the image using the following command:

:::code-group

```bash [Linux]

> docker build -t app_name:latest .

Reviewer: remove this from `Mac w/ Apple Silicon` command too

You can now visit [playground.stf.xyz](https://playground.stf.xyz) in your browser to interact with your micro-rollup. If prompted give it the URL of your local server started by playground.

### Everything else...

You can refer to [Micro-Rollup Utilities](/build/framework/micro-rollup-utils) section for details on all the methods available to interact with the `MicroRollup` instance.

## Next Steps

In this section, we learned how to run our micro-rollup and interact with it.

:::info
You can find the full breakdown of this app in the [Counter Rollup Tutorial](/build/guides/tutorials/counter) guide.
:::

> You can now start building and customizing your own rollup using the Stackr SDK. Start by modifying the `machine.ts`, `transitions.ts` and relevant files to suit your requirements.

Reviewer: ```suggestion
You can start building and customizing your rollup using the Stackr SDK. Start by modifying the `machine.ts`, `transitions.ts`, and relevant files as per your requirements.
```

You can wrap the micro-rollup in a REST API or GraphQL API to interact with it from the frontend.

In the next section, we will learn how to register and deploy our application on Stackr's network and on-chain.

> 

Reviewer: ```suggestion
```
